From 637439277206500bdbbd537c24e188c5ea98642d Mon Sep 17 00:00:00 2001
From: Sveinung Kvilhaugsvik <sveinung84@users.sourceforge.net>
Date: Fri, 24 Jun 2016 02:27:53 +0200
Subject: [PATCH] Freeciv-web: let a ctrl user change scorelog.

Give custom access levels for the scorelog setting in Freeciv-web builds.
All users can see it. A user with the ctrl access level can change it.
Freeciv-web gives the ctrl access level to the first player that connects.
---
 server/settings.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/server/settings.c b/server/settings.c
index 7ba849e..4ef4389 100644
--- a/server/settings.c
+++ b/server/settings.c
@@ -2793,7 +2793,11 @@ static struct setting settings[] = {
 
   GEN_BOOL("scorelog", game.server.scorelog,
            SSET_META, SSET_INTERNAL, SSET_SITUATIONAL,
+#ifdef FREECIV_WEB
+           ALLOW_NONE, ALLOW_CTRL,
+#else /* FREECIV_WEB */
            ALLOW_HACK, ALLOW_HACK,
+#endif /* FREECIV_WEB */
            N_("Whether to log player statistics"),
            /* TRANS: The string between single quotes is a setting name and
             * should not be translated. */
-- 
2.1.4

